tutorial.md

```diff
@@ -0,0 +1,75 @@
+# Windows steps to install Chapel
+
+### Install Ubuntu and initial steps to install chapel 
+    
+    - Create a new folder  
+        - Clone the files from GitHub arkouda (https://github.com/Bears-R-Us/arkouda) and arkouda-njit (https://github.com/Bears-R-Us/arkouda-njit) and save them in a folder called RESEARCH in the C-drive / desktop (anywhere where you will remember the path.) 
+        
+        - Install the chapel-1.28.0.tar.gz file 
+
+    - Install Ubuntu-WSL from this website (https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#2-install-wsl/)
+
+    - Activate/make your Linux account and open the ubuntu terminal 
+
+    - Type bash in the terminal 
+
+       - bash is all set to be used! 
+
+    - Now, Use the commands on the chapel-tutorial link (https://chapel-lang.org/docs/usingchapel/QUICKSTART.html) instructions to unzip the file extract and run chapel on the terminal
```

Comment:
The content is great, however can you please break the steps into smaller blocks of info and code? I checked the preview for this and there was a scroll bar popping up as the sentences were too big to fit

Reply:
I still see the same issue of the scroll bar popping up. That block looks like code?